kolibri/plugins/coach/assets/src/views/recent-page/report-list.vue

```diff
@@ -54,8 +54,8 @@
       'content-icon': require('kolibri.coreVue.components.contentIcon'),
       'progress-bar': require('kolibri.coreVue.components.progressBar'),
     },
-    computed: {
-      reportList() {
+    watch: {
```

Comment:
It's a fair point, it's simpler and very slightly more performant to just sort in the action after receiving from the API and before committing to vuex.

Main reason to keep it in the vue is to keep the actions display-agnostic. E.g. if we ever wanted to update the view to sort by either name or date, these changes would be confined to the display layer.